client/extensions/woocommerce/state/sites/settings/stripe-connect-account/handlers.js

```diff
@@ -55,6 +64,33 @@ export function handleAccountCreateFailure( { dispatch }, action, error ) {
 	} );
 }
 
+export function handleAccountDisconnect( { dispatch }, action ) {
+	const { siteId } = action;
+	dispatch(
+		request( siteId, action, '/wc/v1' ).post( 'connect/stripe/account/deauthorize', {
+			deauthorize: true,
+		} )
+	);
+}
+
+export function handleAccountDisconnectSuccess( store, action ) {
```

Comment:
destructure dispatch as in above and below.

Reply:
Big rewrite in https://github.com/Automattic/wp-calypso/pull/19244/commits/2d81a7a15a8b02918fd6b2ef39b9a54500e0b52c got rid of this